js/constants.js

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+module.exports = {
+  CONTEXT_FILENAME: 'odrl.jsonld',
+  CONTEXT_URL_V1: 'https://www.w3.org/ns/odrl.jsonld'
+};
```

Comment:
Do we need to add this context to the list of registered contexts in cborld and export a cborld value for this context here?

Reply:
No, we don't need to do this. I expect the ODRL context will not be used (at all) as time progresses... we thought the group that created it was going to be more successful than it was. They've since disbanded, so we're probably just going to remove ODRL from all the examples. That said, we need this repo to exist and a v1 release to be made because I need to use this in the W3C VCWG examples.